Missing test data in source distribution

It seems that the 0.5.0 is missing some files needed for tests:

```pytb
FAILED tests/test_check.py::TestIsBinary::test_ds_store - FileNotFoundError: [Errno 2] No such file or directory: 'tests/files/.DS_Stor
e'
FAILED tests/test_check.py::TestProgrammingArtifacts::test_binary_empty_pyc - FileNotFoundError: [Errno 2] No such file or directory: '
tests/files/empty.pyc'
FAILED tests/test_check.py::TestProgrammingArtifacts::test_binary_pyc - FileNotFoundError: [Errno 2] No such file or directory: 'tests/files/hello_world.pyc'
FAILED tests/test_check.py::TestProgrammingArtifacts::test_binary_troublesome_pyc - FileNotFoundError: [Errno 2] No such file or directory: 'tests/files/troublesome.pyc'
```

My educated guess is that hatchling excludes dotfiles and `.pyc` by default.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -100,6 +100,12 @@ exclude_also = [
     "\\.\\.\\.",
 ]
 
+[tool.hatch.build.targets.sdist.force-include]
+"tests/files/.DS_Store" = "tests/files/.DS_Store"
+"tests/files/empty.pyc" = "tests/files/empty.pyc"
+"tests/files/hello_world.pyc" = "tests/files/hello_world.pyc"
+"tests/files/troublesome.pyc" = "tests/files/troublesome.pyc"
+
 [tool.uv]
 package = true
 default-groups = ["dev"]

--- a/tests/test_sdist.py
+++ b/tests/test_sdist.py
@@ -1,0 +1,37 @@
+"""Verify that test fixtures are included in the sdist."""
+
+import subprocess
+import tarfile
+import unittest
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parent.parent
+
+
+class TestSdistContents(unittest.TestCase):
+    def test_sdist_includes_pyc_fixtures(self):
+        """The .pyc test fixtures must survive hatchling's sdist build."""
+        result = subprocess.run(
+            ["uv", "build", "--sdist", "--out-dir", "dist"],
+            cwd=ROOT,
+            capture_output=True,
+            text=True,
+        )
+        self.assertEqual(result.returncode, 0, result.stderr)
+
+        sdists = sorted(ROOT.glob("dist/*.tar.gz"))
+        self.assertTrue(sdists, "No sdist found in dist/")
+        sdist_path = sdists[-1]
+
+        with tarfile.open(sdist_path) as tar:
+            names = tar.getnames()
+
+        expected = [
+            "tests/files/.DS_Store",
+            "tests/files/empty.pyc",
+            "tests/files/hello_world.pyc",
+            "tests/files/troublesome.pyc",
+        ]
+        for fixture in expected:
+            matching = [n for n in names if n.endswith(fixture)]
+            self.assertTrue(matching, f"{fixture} not found in sdist")

